.github/workflows/test_develop.yml

```
@@ -14,12 +14,12 @@ jobs:
   trigger_tests:
     strategy:
       matrix:
-        branch: [develop, release/cycle-3.120, release/cycle-4.1]  # List other branches here
+        branch: [develop, release/cycle-3.120, release/cycle-4.2]  # List other branches here
```

suggestion: we can keep 4.1 until 4.2 is released in case of hotfix
```suggestion
        branch: [develop, release/cycle-3.120,  release/cycle-4.1, release/cycle-4.2]  # List other branches here
```